interface/patient_file/report/custom_report.php

```diff
@@ -45,6 +45,12 @@
 $PDF_OUTPUT = empty($_POST['pdf']) ? 0 : intval($_POST['pdf']);
 
 if ($PDF_OUTPUT) {
+    /* Because of the nature of this report, it is difficult to catch all the various errors possible
+     * so, turning on E_ERROR seems logical to catch memory and php errors that can't be reported due to output buffering.
+     * Try/catch will handle errors after buffer content is retrieved.
+     */
+    ini_set('display_errors', 1); // turn on diswplaying of errors.
+    error_reporting(E_ERROR); // Just fatal php errors are what concern us.
```

Comment:
Would it make sense to encircle above with a global called pdf_html_report_debug which we would leave on in development versions but then turn off in production releases (for example, this is done with the 'dummy' language global)?